addons/60fps/addon.json

```
@@ -28,7 +28,7 @@
   ],
   "settings": [
     {
-      "name": "Alt+GreenFlag FPS",
+      "name": "Custom frames per second (FPS)",
```

Not sure if a setting name really needs to include "frames per second" and its abbreviation. It could be moved into the description, perhaps.
```suggestion
      "name": "Custom frames per second",
```

Or we could use only an abbreviation:
```suggestion
      "name": "Custom FPS",
```